src/utils/wallet.js

```diff
@@ -831,7 +803,7 @@ class Wallet {
             const newKeyPair = KeyPair.fromRandom('ed25519')
             
             try {
-                await this.addAccessKey(accountId, accountId, newKeyPair.publicKey, fromSeedPhraseRecovery)
+                await this.addAccessKey(accountId, accountId, newKeyPair.publicKey, fromSeedPhraseRecovery, '', recoveryKeyIsFAK)
```

Comment:
@mattlockyer not sure I understand the logic. Do we basically add full access key if recovering through seed phrase? Or do I misunderstand something?

Reply:
This means that when you recover with seed phrase we use the seed phrase full access key and not a 2FA tx to add the key

Means seed phrase recovery method can add key without going through 2FA flow